Day5/src/style.scss

```diff
@@ -0,0 +1,130 @@
+$color_1: #334155;
+$background-color_1: #e2e8f0;
+
+.card{
+    max-width: 200px;
+    border: 1px solid black;
+    padding: 10px;
+    margin: 20px;
+    border-radius: 5px;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    &-dp>img{
+        width: 100%;
+        height: 250px;
+        border-radius: 20px;
+    }
+    &-content{
+        padding: 10px;
+    }
+    &:hover {
+        box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+      }
+    &-company{
+        color: #292222 !important;
+        font-weight: 500 !important;
+        padding-top: 0.3rem;
+        text-transform: uppercase;
+      }
+}
+
+.header-container {
+    position: relative;
+    >img {
+        height: 400px;
+        width: 100%;;
+    }
+    &-left{
+        position: absolute;
+        text-align: center;
+    top: 50%;
+    left:50%;
+    font-size: 30px;
+    font-weight: 500;
+    color: $background-color_1;
+    }
+}
+
+.card-container{
+    display: flex;
+    margin: 10px auto;
+    flex-wrap: wrap;
+    max-width: 1440px;
+   justify-content: space-around
+}
+
+input[type="text"]::placeholder {
+  margin-top: 2px;
+  font-size: 1rem;
+}
+
+input[type="text"] {
```

Comment:
can move generic css selectors like html tag selectors to the top to follow CSS specificity.